modules/login/src/main/resources/META-INF/resources/login/login.jsp

```diff
@@ -1,199 +1,375 @@
+                                               data-remote="http://nicsweb:8082/nics/verifyData" required>
+
+                                    </div>
+                                    <div class="help-block with-errors"></div>
+                                </div>
+                            </div>
+                            <div class="col-sm-6">
+                                <!-- New pattern: pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@#$%_!-])[a-zA-Z0-9@#$%_!-]{8,20}$" -->
+                                <!-- Pattern Used in em-api: pattern="((?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%!_-]).{8,20})" -->
+                                <div class="form-group">
+                                    <div class="input-group" style="width:100%">
+                                        <span class="input-group-addon glyphicon glyphicon-lock" style="top: 0"></span>
+                                        <input type="password" id="regpassword"
+                                               class="form-control requiredtext"
+                                               placeholder="Password"
+                                               maxlength="20"
+                                               pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@#$%_!-])[a-zA-Z0-9@#$%_!-]{8,20}$"
+                                               required>
+                                    </div>
+                                    <div class="help-block">8-20 characters with at least one digit,
+                                        one upper case letter, one lower case letter and one special symbol @#$%-_!</div>
+                                </div>
+                            </div>
+                            <div class="col-sm-6">
+                                <div class="form-group">
+                                    <div class="input-group" style="width:100%">
+                                        <span class="input-group-addon glyphicon glyphicon-lock" style="top: 0"></span>
+                                        <input type="password" id="confirmpregassword"
+                                               class="form-control requiredtext"
+                                               placeholder="Confirm Password"
+                                               data-match="#regpassword"
+                                               maxlength="20"
+                                               data-match-error="Passwords don't match." required>
+
+                                    </div>
+                                    <div class="help-block with-errors"></div>
+                                </div>
+                            </div>
+
+                        </div>
+
+                    </div>
+                    <div class="modal-footer">
+
+                        <button id="btnPrev2" type="button" class="btn btn-primary" onclick="registration.doStep1()"><< Previous</button>
```

Comment:
Consider using ```&lt;``` entity code here and the other places where you want literal ```<``` output.